src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -65,7 +65,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'YOUR_INFORMAT
             </div>
           </div>
         </div>
-        <div class="col-sm-2">
+        <div ng-if="$ctrl.useV3 !== 'true'" class="col-sm-2">
```

Comment:
The CSS should probably be updated to make first/last name look nicer in this case.

<img width="913" alt="Screenshot 2024-12-18 at 2 04 21 PM" src="https://github.com/user-attachments/assets/78b8ce22-6882-4cc2-b0d6-ebe0e2e4e4fb" />